<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Blocks</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="blocksFilterE">
          <property name="placeholderText">
           <string>Filtr wyszukiwania</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BlocksTree" name="blocksW">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="indentation">
           <number>15</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Źródła</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Przetwarzanie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Wyjścia</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="SchemeView" name="graphicsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Pasek narzędzi</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNowy"/>
   <addaction name="separator"/>
   <addaction name="actionOtw_rz"/>
   <addaction name="actionZapisz"/>
   <addaction name="actionZapisz_jako"/>
   <addaction name="separator"/>
   <addaction name="actionCofnij"/>
   <addaction name="actionPrzywr_c"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="action_Ustawienia"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Plik">
    <property name="title">
     <string>&amp;Plik</string>
    </property>
    <addaction name="actionNowy"/>
    <addaction name="separator"/>
    <addaction name="actionOtw_rz"/>
    <addaction name="actionZapisz"/>
    <addaction name="actionZapisz_jako"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menu_Edycja">
    <property name="title">
     <string>&amp;Edycja</string>
    </property>
    <addaction name="actionCofnij"/>
    <addaction name="actionPrzywr_c"/>
   </widget>
   <widget class="QMenu" name="menu_Pomoc">
    <property name="title">
     <string>&amp;Pomoc</string>
    </property>
    <addaction name="action_O_programie"/>
    <addaction name="actionO_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Symulacja">
    <property name="title">
     <string>&amp;Symulacja</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="action_Ustawienia"/>
   </widget>
   <widget class="QMenu" name="menu_Dodatki">
    <property name="title">
     <string>&amp;Dodatki</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Widok">
    <property name="title">
     <string>&amp;Widok</string>
    </property>
    <addaction name="action_U_yj_natywnych_okien_dialogowych"/>
   </widget>
   <addaction name="menu_Plik"/>
   <addaction name="menu_Edycja"/>
   <addaction name="menu_Symulacja"/>
   <addaction name="menu_Dodatki"/>
   <addaction name="menu_Widok"/>
   <addaction name="menu_Pomoc"/>
  </widget>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOtw_rz">
   <property name="text">
    <string>Otwórz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNowy">
   <property name="text">
    <string>Nowy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCofnij">
   <property name="text">
    <string>Cofnij</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionPrzywr_c">
   <property name="text">
    <string>Przywróć</string>
   </property>
   <property name="toolTip">
    <string>Przywróć</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="action_O_programie">
   <property name="text">
    <string>&amp;O programie</string>
   </property>
  </action>
  <action name="actionO_Qt">
   <property name="text">
    <string>O &amp;Qt</string>
   </property>
  </action>
  <action name="actionZapisz_jako">
   <property name="text">
    <string>Zapisz jako</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_Ustawienia">
   <property name="text">
    <string>&amp;Ustawienia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_U_yj_natywnych_okien_dialogowych">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Użyj natywnych okien dialogowych</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BlocksTree</class>
   <extends>QTreeWidget</extends>
   <header>BlocksTree.hpp</header>
  </customwidget>
  <customwidget>
   <class>SchemeView</class>
   <extends>QGraphicsView</extends>
   <header>SchemeView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>blocksFilterE</tabstop>
  <tabstop>blocksW</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionZamknij</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
